pom.xml

```diff
@@ -29,6 +29,7 @@
 		<module>msdk-id-metfrag</module>
 		<module>msdk-id-sirius</module>
 		<module>msdk-io-filetypedetection</module>
+		<module>msdk-io-mz5</module>
```

Comment:
Perhaps remove this?

Reply:
Sorry, somehow progress from my other branch stayed to this while switching branches. I removed the module folder from this branch, but, forgot to remove it from pom.xml